src/reducers/field008.js

```diff
@@ -115,25 +134,82 @@ function process008(base, source) {
   // Copy missing 008 from source (theoretical)
   if (!base008) { // eslint-disable-line functional/no-conditional-statements
     const clonedField = clone(source008);
-    base.insertField(clonedField); // Should we clone this?
+    base.insertField(clonedField);
     return;
   }
 
+  //console.info(`B: '${base008.value}'\nS: '${source008.value}'`); // eslint-disable-line no-console
+
   // Switch fields it base has illegal length:
   if (base008.value.length !== 40) {
     base008.value = source008.value; // eslint-disable-line functional/immutable-data
     return;
   }
 
+  //console.info(`${base008.value}\n${source008.value}`); // eslint-disable-line no-console
 
+  // All materials (008/00-17, and 008/35-39)
   setOlderDateToBase(base008, source008); // 008/00-05
-
   setDates(base008, source008); // 008/06,07-10,11-14
   setPlaceOfPublication(base008, source008); // 008/15-17
 
   setLanguage(base008, source008); // 008/35-37
-
   setCatalogingSource(base008, source008); // 008/39
+  // Type of material specific code:
+  const typeOfMaterial = base.getTypeOfMaterial(); // Boldly assume that type of material is same for both...
```

Comment:
Let's check both?

Reply:
yes, my bad. done.

Reply:
Implementation changed. Requested functionality is there